cmd/init.go

```diff
@@ -43,6 +44,112 @@ func (h *Handler) initNew(ctx context.Context, req *entity.CommandRequest) error
 	return nil
 }
 
+func (h *Handler) initFromTemplate(ctx context.Context, req *entity.CommandRequest) error {
+	// Prompt for a template to use
+
+	template, err := ui.PromptStarterTemplates()
+	if err != nil {
+		return err
+	}
+
+	// Select GitHub org
+
+	ui.StartSpinner(&ui.SpinnerCfg{Message: "Fetching GitHub scopes"})
+	scopes, err := h.ctrl.GetWritableGithubScopes(ctx)
+	if err != nil {
+		return err
+	}
+	if len(scopes) == 0 {
+		return errors2.NoGitHubScopesFound
+	}
+	ui.StopSpinner("")
+
+	org, err := ui.PromptGitHubScopes(scopes)
+	if err != nil {
+		return err
+	}
```

Comment:
Thoughts on, in the event of a single scope, just selecting it for the user? IMO, the less steps the better
```suggestion
        if len(scopes) == 1 {
            fmt.Printf("Using %s GitHub Account\n", scopes[0])
        } else {
            org, err := ui.PromptGitHubScopes(scopes)
	    if err != nil {
	        return err
	    }
        }
```

Reply:
This logic is in `ui.PromptGitHubScopes`

Reply:
Oh perfecto